tests/test_wave_03.py

```diff
@@ -22,8 +22,10 @@ def test_swap_items_returns_true():
     jolie = Vendor(
         inventory=[item_d, item_e]
     )
-
+    
+    print(fatimah.inventory)
```

Comment:
Just a little thing, but its preferred that these little changed aren't added to the commits. This can be avoided by using `git add` with specific file names instead of typing `git add .`